src/core/types/array.js

```diff
@@ -14,6 +14,34 @@
  * limitations under the License.
  */
 
+/**
+ * Converts an array-like object to an array.
+ * @param {?IArrayLike<T>|string} arrayLike
+ * @return {!Array<T>}
+ * @template T
+ */
+export const toArray = Array.prototype.slice.call;
```

Comment:
This was previously
```javascript
export function toArray(arrayLike) {
  return arrayLike ? Array.prototype.slice.call(arrayLike) : [];
}
```
However `Array.prototype.slice.call(anyFalseyValue)` returns `[]` regardless, so AFAICT the extra logic and wrapping function may be unnecessary.